Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.m

```diff
@@ -214,22 +225,33 @@ - (void)setupWithOptions:(NSArray<BPKHorizontalNavigationOption *> *)options sel
     self.barView.hidden = YES;
     [self addSubview:self.barView];
 
-    self.barConstraints = @[
-        [self.barView.leadingAnchor constraintEqualToAnchor:self.leadingAnchor],
-        [self.barView.trailingAnchor constraintEqualToAnchor:self.leadingAnchor]
-    ];
-    [NSLayoutConstraint activateConstraints:self.barConstraints];
+    self.barTopConstraint = [NSLayoutConstraint constraintWithItem:self.barView
```

Comment:
We tend to use these using the following syntax:
```
[self.barView.topAnchor constrain...];
```

Maybe worth keeping it consistent?